src/levels.rs

```diff
@@ -55,7 +55,6 @@ pub enum TxfmSize {
 
 impl TxfmSize {
     pub const NUM_SQUARE: usize = Self::S64x64 as usize + 1;
-    pub const NUM_RECT: usize = Self::COUNT;
```

Comment:
Can we leave this with just a `_` prefix? Since a similar constant is in C, so it could be used later.  These are also things I think we'll make public for the crate if we add a Rust API, so I don't think we want to just completely delete it.